src/main/java/org/kainos/ea/cli/Job.java

```diff
@@ -0,0 +1,57 @@
+package org.kainos.ea.cli;
+
+public class Job {
```

Comment:
Does this represent a `Job` or a `JobSpec` as it's been called `Job`, but when used it's being called `JobSpec`

Reply:
So I am pulling the spec field from the Job Object - because Im using the JobName etc on Job Spec UI page